packages/inter-protocol/src/psm/psm.js

```diff
@@ -292,17 +308,23 @@ export const prepare = async (zcf, privateArgs, baggage) => {
     M.interface('PSM', {
       getMetrics: M.call().returns(M.remotable('MetricsSubscriber')),
       getPoolBalance: M.call().returns(anchorAmountShape),
+      getPublicTopics: M.call().returns({
+        metrics: M.any(),
```

Comment:
`M.record()` maybe?

Reply:
thanks for pointing out the any. there's actually a `TopicsRecordShape` already so I added that